README.md

```diff
@@ -57,6 +57,23 @@ The tooling has evolved over time and therefore when generating new packages, yo
 This is because the new package has a dependency on an existing package, it was regenerated and changes were detected from
 when it was originally generated.
 
+#### Backends
+The tooling support two backends for generating reference assemblies: [the Cci-based GenAPI](https://github.com/dotnet/arcade/tree/main/src/Microsoft.DotNet.GenAPI) and [the Roslyn-based GenAPI](https://github.com/dotnet/sdk/tree/main/src/GenAPI). The default one is Roslyn-based. You can switch between backends using the `--genapi-backend` parameter that supports `cci` and `roslyn` values.
```

Comment:
Per our discussion last week, I thought we were going all in on the Roslyn backend.  I don't think the `CCI` backend is even worth documenting in this readme.  We can continue to have to option but it would be an edge case to use it.  Because of that, I don't want to call it to attention as it is a distraction.  Let's keep the documentation to the canonical use case and therefore keep it simple.

Reply:
> Are you planning to update the default parameter with this PR? This says the default is roslyn but on the tip it is cci.

Yes, added changes.

Reply:
We are going to have cci based as a fallback solution for some period of time. Let's say month-two and then remove it as a result from the documentation

Reply:
> > Are you planning to update the default parameter with this PR? This says the default is roslyn but on the tip it is cci.
> 
> Yes, added changes.

I am going to merge it no sooner Thuesday.